build.gradle

```diff
@@ -16,6 +16,10 @@ plugins {
 
 repositories {
     mavenCentral()
+
+    maven {
+        url "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/" // for intel deflater snapshots
```

Comment:
The classes we use in `intel-gkl` implement interfaces from `gatk-native-bindings`, so I don't think we can exclude it.